data-access/src/infrastructure-services-impl/datastore/mongodb/models/community.ts

```diff
@@ -25,6 +40,7 @@ export const CommunityModel = model<Community>('Community',new Schema<Community,
       required: false, 
       maxlength: 50,
     },
+    approvedVendors: { type: [approvedVendorSchema], required: false },
```

Comment:
Change the type of approvedVendors to look like it does below. Use VendorModel instead of EntityModel
![image](https://github.com/user-attachments/assets/7e45ba98-efd3-4629-8e16-dc8b04d0a49c)